src/lib/mina_ledger/ledger.ml

```diff
@@ -529,9 +529,8 @@ let%test_unit "tokens test" =
                (-(4 * account_creation_fee)) )
             []
         ; mk_node
-            (mk_account_update_body
-               (Proof (Zkapp_account.dummy_vk_hash ()))
-               Call token_owner Token_id.default (3 * account_creation_fee) )
+            (mk_account_update_body Signature Call token_owner Token_id.default
```

Comment:
There's a bug in this test, if we use `Proof`, because the corresponding ledger account has `None` for the `zkapp` field, hence no vk hash, so the new vk hash check fails.

Reply:
I'm able to patch the ledger so the token funder has a zkapp with the dummy verification key, allowing `Proof` for this sub-test. (It would require a bit more change to do the same for the token owner.)